templates/CRM/Contact/Import/Form/MapField.tpl

```diff
@@ -29,8 +29,7 @@
         $('#map-field').on('change', 'select[id^="mapper"][id$="_0"]', function() {
           var select = $(this).next();
           $('option', select).each(function() {
-            if ($(this).attr('value') == defaultLocationType  && $(this).text() == {/literal}{
-              $defaultLocationTypeLabel|@json_encode}{literal}) {
+            if ($(this).attr('value') == defaultLocationType  && $(this).text() == {/literal}{$defaultLocationTypeLabel|@json_encode}{literal}) {
```

Comment:
In Smarty 2, `{\n$foo`  will evaluate `$foo`. But in Smarty 4/5, it does not.

Reply:
It's a pretty bad line of code generally. The variable should be getting assigned along with `var defaultLocationType`, not in the middle of an if condition.